### PR TITLE
NO-ISSUE: sonata flow deployment finalizations

### PR DESCRIPTION
**Pull request by fantonangeli**

### CODE REVIEW

File: packages/sonataflow-deployment-webapp/webpack.config.js [hunk 1]
```diff
@@ -32,12 +33,26 @@ module.exports = async (env) =>
     entry: {
       index: "./src/index.tsx",
     },
+    optimization: {
+      minimize: true, // This should already be there
```
Comment (caponetto) [inline packages/sonataflow-deployment-webapp/webpack.config.js]: Do we need this line? We already have this prop in `webpack.common.config.js`

File: packages/sonataflow-deployment-webapp/README.md [hunk 2]
```diff
@@ -78,7 +79,7 @@ To create a consumer app for local tests following the Quarkus Getting Started A
 Add the webjar version to the properties section of your pom.xml
 
 ```xml
-<slwtDeploymentWebapp.version>0.0.2</slwtDeploymentWebapp.version>
+<sonataFlowDeploymentWebapp.version>0.0.2</sonataFlowDeploymentWebapp.version>
```
Comment (caponetto) [inline packages/sonataflow-deployment-webapp/README.md]:
```suggestion
<sonataFlowDeploymentWebapp.version>0.32.0</sonataFlowDeploymentWebapp.version>
```
We should not keep updating this version but let's use the first version that will possibly be out.
Also, I think we should instruct the user to go to webjars website and make sure to use the latest version.

File: packages/sonataflow-deployment-webapp/README.md [hunk 3]
```diff
@@ -39,7 +39,8 @@ To create a consumer app for local tests following the Quarkus Getting Started A
 
 ### Using a local webjar
 
-1.  Edit the `pom.xml` file and add the following plugin configuration:
+1.  From this project run `mvn install`
```
Comment (caponetto) [inline packages/sonataflow-deployment-webapp/README.md]: Why this is necessary?

Comment (fantonangeli) [reply packages/sonataflow-deployment-webapp/README.md]: `mvn install` builds the project and installs it in the local `.m2` repository.

Comment (caponetto) [reply packages/sonataflow-deployment-webapp/README.md]: Yes, I know. But why is that necessary to be in the instruction?

Comment (fantonangeli) [reply packages/sonataflow-deployment-webapp/README.md]: it's necessary to have it available in the `.m2` repo, to use it in the consumer app

Comment (caponetto) [reply packages/sonataflow-deployment-webapp/README.md]: Got it, thanks!
Can you include this info in the step?
Something like:
```suggestion
1.  From this project run `mvn install` to have it available in the `.m2` repository so you can use it in your consumer app
```

**Review by caponetto — APPROVED** [review]

**Pull request merged**